universal-application-tool-0.0.1/app/auth/Roles.java

```diff
@@ -0,0 +1,8 @@
+package auth;
+
+public class Roles {
+  public static String ROLE_APPLICANT = "ROLE_APPLICANT";
+  public static String ROLE_PROGRAM_ADMIN = "ROLE_PROGRAM_ADMIN";
+  public static String ROLE_TI = "ROLE_TI";
+  public static String ROLE_UAT_ADMIN = "ROLE_UAT_ADMIN";
+}
```

Comment:
Suggestion (blocking): I think this would be better expressed as an enum:

```java
public enum Roles {
  ROLE_APPLICANT("ROLE_APPLICANT"),
  ROLE_PROGRAM_ADMIN("ROLE_PROGRAM_ADMIN"); // etc
  
  private final String roleName;

  Roles(String roleName) {
    this.roleName = roleName;
  }

    @Override
    public String toString() {
        return roleName;
    }  
}
```

Reply:
Haha I forgot that enums even existed - python and go exclusively for the last 4 years.  Of course.